src/modules/transaction/hooks/useTransactionFee/useTransactionFee.test.js

```diff
@@ -0,0 +1,83 @@
+import { renderHook } from '@testing-library/react-hooks';
+import wallets from '@tests/constants/wallets';
+import { mockAuth } from '@auth/__fixtures__';
+import { useAuth } from '@auth/hooks/queries';
+import { useTransactionFee } from './useTransactionFee';
+
+const bufferify = (string) => Buffer.from(string, 'hex');
+
+const defaultPriorities = [
+  { value: 0, title: 'Low' },
+  { value: 1, title: 'Medium' },
+  { value: 2, title: 'High' },
+];
+const transaction = {
+  nonce: BigInt(0),
+  senderPublicKey: Buffer.from(wallets.genesis.summary.publicKey, 'hex'),
+  module: 'token',
+  command: 'transfer',
+  params: {
+    tokenId: bufferify('00000000'),
+    amount: BigInt('1000000'),
+    recipientAddress: bufferify(wallets.genesis.summary.address),
+    data: '',
+  },
+  signatures: [],
+};
+
+jest.mock('@auth/hooks/queries');
+jest.mock('@network/hooks', () => ({
```

Comment:
useAuth is already mocked with msw please avoid moc it on this way by the way if you need to please use Jest.spyOn then mock it, mock entier directory it will efect other test using this hook